plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -576,6 +576,9 @@ export class EventsProcessor {
 
         const elementsChain = elements && elements.length ? elementsToString(elements) : ''
 
+        // TODO: don't parse back and forth with json
```

Comment:
let's then just add a flag `getRawProperties` or something. It can event be a separate value in the return object

Reply:
Don't even need that - I can just make it return strings as I'll be writing them back as strings anyway